nautobot/docs/plugins/development.md

```diff
@@ -1090,7 +1090,6 @@ Other context keys may be available for certain views:
 
 You may see other context keys as well, but any not documented above should not be relied upon as they may be removed in a future release. Some examples of those are:
 
-* `changelog_url`: This can now be retrieved from the object itself, via `object.get_changelog_url`, if the object supports change-logging
```

Comment:
Possibly add a version-changed admonition  below this list, something like:

```markdown
+/- 2.0.0
    The `changelog_url` context key was removed. Use `object.get_changelog_url` instead.
```

Reply:
What about `---` since it was removed?